docs/usage/macros.rst

```diff
@@ -4,44 +4,67 @@ CMake Macros
 spl_add_component
 ^^^^^^^^^^^^^^^^^
 
-Add a component to a variant.
+Add a component to the build and link it with the main project.
 
 .. code-block:: cmake
 
-  spl_add_component(<relative path from project root to the component's directory>)
+ spl_add_component(<component_path>)
```

Comment:
Mmmmh, the description before was clear. The relative path from the project root to the component's directory. At least the relative path must be mentioned.